Remotion/Web/ClientScript/Scripts/DatePicker.ts

```diff
@@ -113,25 +122,25 @@ class DatePicker
 
     DatePicker.CloseDatePicker(true);
 
-    var isValueChanged = current.Target.val() != value;
+    var isValueChanged = current.Target.value != value;
     if (isValueChanged)
     {
-      current.Target.val(value);
-      current.Target.change();
+      current.Target.value = value;
+      current.Target.dispatchEvent(new Event('change'));
```

Comment:
Helper methods for registering/unregistering do not make sense but helper methods for dispatching might make sense.

Reply:
I was thinking since we're using string constants for the events, pre-defining those makes sense.

Reply:
Pre-defining the names does not make a lot of sense because some events require a different event object. When registering/unregistering the events with `addEventListener`/`removeEventListener` it also does not make sense to use pre-defined event names because the names are already registered as an "overload" to determine the correct event handler type.

The solution that makes the most sense would be to create a "EventFactory" with methods for each event that is needed. This way you do not have to care about event object + event name. But I also think that the benefit of such a solution is not that big compared to the additional boilerplate it introduces.